### PR TITLE
create translation files (german)

### PR DESCRIPTION
**Pull request by rschnapka**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: account_direct_debit/migrations/7.0.2/pre-migration.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def rename_columns(cr, column_spec):
     for table in column_spec.keys():
         for (old, new) in column_spec[table]:
             logger.info("table %s, column %s: renaming to %s", table, old, new)
-            cr.execute('ALTER TABLE %s RENAME %s TO %s', (table, old, new,))
+            cr.execute('ALTER TABLE %s RENAME %s TO %s' % (table, old, new,))
```
Comment (yvaucher) [inline account_direct_debit/migrations/7.0.2/pre-migration.py]: What is this change is this diff? It has nothing to do with translations

**Comment by rschnapka** [comment]
oops, seems that change slipped through. In the verge of testing I ran over this proposal 'fix' and put it in - seemed correct to me. (Yes: It has nothing to do with translations, correct)
Should I revert that change?

**Comment by yvaucher** [comment]
@rschnapka Yes please remove it. You can squash it from your previous commits

**Comment by rschnapka** [comment]
sorry i am still not so affirmed with all those nice terms (like squashing). I am still getting accustomed to the nice possibilities of git (and versioning in general). For now I just committed the reversal of the change as I could not figure out quickly how to revert a single change out of the whole commit - hope that's ok :-)

**Comment by yvaucher** [comment]
@rschnapka You could do 

```
git rebase -i
```

This allows you to edit your commits to make it cleaner.

As you added this in 4th commit, in the opened editor (vim by default) you can replace "pick" by "fixup"

Result should be as:

```
pick commithash1
pick commithash2
pick commithash3
pick commithash4
fixup commithash5
```

This will merge commit 5 into commit 4

**Comment by yvaucher** [comment]
@rschnapka An other option when you fix last commit is to do your change and then do a 

```
git commit --amend
```

This will add your changes in last commit and ask you if you want to edit commit message.

**Comment by yvaucher** [comment]
I made the commit amend in https://github.com/OCA/bank-payment/pull/160

**Comment by yvaucher** [comment]
Merged

**Pull request closed without merging**